Assets/MixedRealityToolkit/Utilities/PlatformUtility.cs

```diff
@@ -20,22 +20,28 @@ private static SupportedPlatforms GetSupportedPlatformMask(RuntimePlatform runti
             switch (runtimePlatform)
             {
                 case RuntimePlatform.WindowsPlayer:
-                case RuntimePlatform.WindowsEditor:
                     supportedPlatforms |= SupportedPlatforms.WindowsStandalone;
                     break;
+                case RuntimePlatform.WindowsEditor:
```

Comment:
I think this change will break most existing services, which don't explicitly mark the editor as a supported platform (like WMRDeviceManager) when running in the editor:

https://github.com/microsoft/MixedRealityToolkit-Unity/blob/mrtk_release/Assets/MixedRealityToolkit.Providers/WindowsMixedReality/WindowsMixedRealityDeviceManager.cs#L26

Reply:
There may be some more changes needed here